### PR TITLE
DOCS: Remove references to Atlassian for LocalStack

### LINKED ISSUE #4793: Upgrade LocalStack module
## Description

Hi 👋 

Testcontainers is currently using an old version of LocalStack and an incompatible configuration setup here: [LocalStackContainer.java](https://github.com/testcontainers/testcontainers-java/blob/master/modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java).

This also corresponds with some of the missing features which is further described on this issue: https://github.com/testcontainers/testcontainers-java/issues/3785

It would be ideal to upgrade the LocalStack module for a wider use-case. P.S: LocalStack is no longer under Atlassian Labs, and is a company ([localstack.cloud](https://localstack.cloud)). We would ideally like the inline documentation to be updated as well here 😄

### PR DESCRIPTION
**Pull request by HarshCasper**
Fixes #4793

### CODE REVIEW

File: docs/modules/localstack.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Localstack Module
 
-Testcontainers module for the Atlassian's [LocalStack](https://github.com/localstack/localstack), 'a fully functional local AWS cloud stack'.
+Testcontainers module for [LocalStack](http://localstack.cloud/), 'a fully functional local AWS cloud stack', to develop and test your cloud and serverless apps offline.
```
Comment (kiview) [inline docs/modules/localstack.md:3]:
```suggestion
Testcontainers module for [LocalStack](http://localstack.cloud/), 'a fully functional local AWS cloud stack', to develop and test your cloud and serverless apps without actually using the cloud.
```

Maybe rephrase like this? Since e.g. the container downloads certificates on startup (something I learned through the recent incident with the expired certificate), so you still need to be online.

File: docs/modules/localstack.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Localstack Module
```
Comment (kiview) [inline docs/modules/localstack.md:1]:
```suggestion
# LocalStack Module
```

File: modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java [hunk 3]
```diff
@@ -25,7 +25,7 @@
 import java.util.stream.Collectors;
 
 /**
- * <p>Container for Atlassian Labs Localstack, 'A fully functional local AWS cloud stack'.</p>
+ * <p>Container for Localstack, 'A fully functional local AWS cloud stack'.</p>
```
Comment (kiview) [inline modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java:28]:
```suggestion
 * <p>Container for LocalStack, 'A fully functional local AWS cloud stack'.</p>
```

Seems we are a bit inconsistent with the naming in our docs, so let's fix it as part of this PR 🙂

**Review by kiview — DISMISSED** [review]

**Review by rnorth — APPROVED** [review]
